src/workflow/workflowOptions.ts

```diff
@@ -74,8 +75,46 @@ export const workflowOptions = {
     ],
     initial: 0,
   },
+  useSurf: {
+    type: (prev, values) => needSurfChoice(values),
+    name: "useSurf",
+    message:
+      "Choose if you want to use Surf, the TypeScript type safety tool maintained by Thala Labs",
+    choices(prev, values) {
```

Comment:
wouldn't be easier to
```
if(values.template.path === FullstackBoilerplateTemplateInfo.value.path){
return [
          {
            title: "Do not use Surf and handle types manually",
            value: false,
          },
          {
            title:
              "Use Surf to auto generate TypeScript Types for your Move contracts",
            value: true,
          },
        ];
}else{
return [
          {
            title: "Do not use Surf and handle types manually",
            value: false,
          },
        ]
}
```

Reply:
surf doesn;t apply to contract only template

Reply:
correct, and here we are only applying to when the template path is `FullstackBoilerplateTemplateInfo.value.path`, these templates use different path https://github.com/aptos-labs/create-aptos-dapp/blob/main/src/utils/constants.ts#L33-L41